src/components/EntityPage/Participant/ParticipantClinical.js

```diff
@@ -143,11 +152,11 @@ class ParticipantClinical extends React.Component {
       { Header: 'Diagnosis (Source Text)', accessor: 'source_text_diagnosis', Cell: cellBreak },
       { Header: 'Age at event', accessor: 'age_at_event_days', Cell: cellBreak },
       {
-        Header: 'Shared with',
+        Header: 'Mondo term shared with',
         accessor: 'shared_with',
         Cell: wrapper => {
 
-          if(wrapper.value === "0"|| wrapper.value === 0) return <div>0</div>;
+          if(wrapper.value === "0 participant") return <div>0 participant</div>;
```

Comment:
In english, we have to write `0 participants`

https://english.stackexchange.com/a/13075/89065